Mathlib/Data/NNRat/Defs.lean

```diff
@@ -30,6 +30,16 @@ Whenever you state a lemma about the coercion `ℚ≥0 → ℚ`, check that Lean
 `Subtype.val`. Else your lemma will never apply.
 -/
 
+library_note "specialised high prio simp lemma" /--
+It sometimes happens that a simp lemma declared early in the library can be proved by `simp` using
+later, more general simp lemmas. In that case, the following reasons might be arguments for the
+early lemma to be tagged `@[simp high]` (rather than `@[simp, nolint simpNF]` or unsimped):
+1. There is a significant portion of the library which needs the early lemma to be available via
+  `simp` and which doesn't have access to the more general lemmas.
+2. The more general lemmas have more complicated typeclass assumptions, causing rewrites with them
+  to be slower.
```

Comment:
```suggestion
1. There is a significant portion of the library which needs the early lemma to be available via
   `simp` and which doesn't have access to the more general lemmas.
2. The more general lemmas have more complicated typeclass assumptions, causing rewrites with them
   to be slower.
```

Reply:
Uh, that doesn't respect the "indent by two spaces" rule. We don't right-flush anything else, eg in `calc` syntax

Reply:
Ultimately what matters is whether our markdown formatter allows this.

... which I guess also doesn't matter because `library_note`s don't yet render anywhere:.